### PR TITLE
MCH: add async post-processing

### PR DESCRIPTION
**Pull request by aferrero2707**
The PR adds the post-processing and checkers for the Digits and Preclusters QC tasks.

The code is the same that is used for the online processing. The PP tasks and the checkers should process the output of the ASYNC QC for each run, producing additional plots in the `MCH/Digits` and `MCH/Preclusters` folders, as well as QualityObjects with the check results in `MCH/QO`

### CODE REVIEW

**Comment by aferrero2707** [comment]
@aphecetche @knopers8 could you please review the changes? I have tested them locally and all seems to work as expected, but I have no experience with the ASYNC processing.Thanks!

**Comment by aphecetche** [comment]
LGTM

**Review by knopers8 — COMMENTED** [review]
Sorry, I missed it before. It look OK, i would only propose to remove "Check" in Check names, as mentioned in the naming convention:
https://github.com/AliceO2Group/QualityControl/blob/master/doc/ModulesDevelopment.md#naming-convention

File: DATA/production/o2dpg_qc_postproc_workflow.py [hunk 1]
```diff
@@ -79,6 +79,7 @@ def add_QC_postprocessing(taskName, qcConfigPath, needs, runSpecific, periodSpec
   ## The list of QC Post-processing workflows, add the new ones below
   add_QC_postprocessing('example', 'json://${O2DPG_ROOT}/DATA/production/qc-postproc-async/example.json', needs=[], runSpecific=False, periodSpecific=False, passSpecific=True)
   add_QC_postprocessing('EMC', 'json://${O2DPG_ROOT}/DATA/production/qc-postproc-async/emc.json', needs=[], runSpecific=False, periodSpecific=True, passSpecific=True)
+  add_QC_postprocessing('MCH', 'json://${O2DPG_ROOT}/DATA/production/qc-postproc-async/mch.json', needs=[], runSpecific=True, periodSpecific=True, passSpecific=True)
```
Comment (knopers8) [inline DATA/production/o2dpg_qc_postproc_workflow.py:82]: Am I correct to understand that you expect to produce an output for each run separately? I.e. you are not making a trend across runs?

**Pull request merged**